docs/api-reference/mcp.md

```
@@ -0,0 +1,226 @@
+
+**Operation ID:** `get_server_endpoint_api_v2_mcp_servers__server_name__get`
+
+
+### Parameters
+
+| Name | Located In | Type | Required | Description |
+|------|-----------|------|----------|-------------|
+| `server_name` | **path** | string | ✅ Yes | - |
+
+
+
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)